field_utils/mapl_udunits2status.F90

```diff
@@ -0,0 +1,28 @@
+! Status values for udunits2 procedures
+! The values are the same as the udunits2 utStatus C enum
+module mapl_udunits2status
+
+   implicit none
+
+   enum, bind(c)
+       enumerator :: &
+       UT_SUCCESS = 0, & ! Success
+       UT_BAD_ARG, & ! An argument violates the function's contract
+       UT_EXISTS, & ! Unit, prefix, or identifier already exists
```

Comment:
Q:   Are you certain that the C layer does not play any funny games with the values of the enums?   I always get nervous in this context because the "coincidence" in the ordering is fragile under change.

If the C values are defined with the preprocessor we could possible directly assign values.. E.g.,
``` 
UT_BAD_ARG = c_ut_bad_arg
```
Means that the Fortran name would need to be spelled differently (or at least capitalized differently.

Reply:
I agree. I copied and pasted from `udunits2` header file(s?). From what I understand, Fortran enums are not really that useful. I thought about making these regular `integer(c_int)` constants.

The C values are in a C typedef enum `ut_status` in a udunits2 header file with `UT_STATUS = 0` and the rest following in the order I used.

With udunits2 in BASELIBS, we can't access the header files. The preprocessor could parse those header files for the `ut_status` values and generate constants, but we would need a copy of the header files included with MAPL, which means we would basically be back where we started, since we wouldn't receive updates for the values. Ugh!

Reply:
ok